src/components/ListOrgs.vue

```diff
@@ -272,6 +276,70 @@ const exportAll = async () => {
   exportCsv(exportData, `roar-${activeOrgType.value}.csv`);
 };
 
+const exportOrgUsers = async (orgType) => {
+  try {
+    // First, count the users
+    const userCount = await countUsersByOrg(activeOrgType.value, orgType.id, orderBy);
+
+    if (userCount === 0) {
+      toast.add({
+        severity: 'error',
+        summary: 'Export Failed',
+        detail: 'No users found for the organization.',
+        life: 3000,
+      });
+      return;
+    }
+
+    if (userCount >= CSV_EXPORT_MAX_RECORD_COUNT) {
```

Comment:
```suggestion
    if (userCount > CSV_EXPORT_MAX_RECORD_COUNT) {
```

A tiny detail, not important, just might be easier to state that we allow exports up to 10.000 users, not up to 9.999 users